hpi-swt2/lecture-portal
# Feature/#84 file type

# Resolves #84

This PR adds the feature of Setting a file type.

Acceptance Criteria:
- [ ] In the DB scheme, a file can be classified as ONE of the following: Notes, Summary, Lecture Slides, Lecture Marerial

- [ ] The scheme should support the future addition of other classifications

Testing steps:
- got to /uploaded_files/new
- enter a valid lecture id
- 
Steps needed for migration:
- first migration step
- second migration step
- add an attachment
- select a file type from the select box
- send the form
- you should see the uploaded file in the table, with the selected File type in the column with this name.

---

Definition Of Done:
